doc/lightning-sendpsbt.7.md

```diff
@@ -21,7 +21,7 @@ EXAMPLE JSON REQUEST
   "id": 82,
   "method": "sendpsbt",
   "params": {
-    "psbt": "some_psbt"
+    "psbt": "cHNidP8BAHEBAAAA..."
```

Comment:
Your other examples used a full string, not a "..." truncated one, seems IMO better to use a full string example.